.gitignore

```diff
@@ -3,4 +3,5 @@
 /coverage.xml
 /.phpcs-cache
 /.phpunit.result.cache
-/composer.lock
\ No newline at end of file
+/composer.lock
+.idea/*
```

Comment:
IMO it should be put in your [Global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c).

Reply:
Porbably, i will remove revert it.